core/src/main/scala/org/apache/spark/storage/DiskBlockObjectWriter.scala

```diff
@@ -65,7 +65,16 @@ private[spark] class DiskBlockObjectWriter(
     }
 
     def manualClose(): Unit = {
```

Comment:
Ideally, we should handle this in `closeResources` - but it is fine here as well, given this is not `close` but `manualClose` and we control the semantics of it :-)